### DISCUSSION TITLE
Add tests for arm64 platform using GA

### DISCUSSION THREAD

**Discussion by odidev** [dexidp/dex#2318] [Q&A]
Hi Team, 

I am working with dexidp/dex Amd64 and Arm64 docker images. 

It would be great if the docker images are tested via CI. Do you have any plans for testing these docker images of arm64 and amd64 via GA? Please let me know if there is any help required from my end, will be happy to contribute.

**Comment by sagikazarmark** [comment]
Hey @odidev !

We plan to run tests on arm64 as well.

What tests do you have in mind for Docker images?

**Reply by odidev** [reply]
We can run the same tests for arm64 as well which we are running for the amd64 platform.

**Reply by sagikazarmark** [reply]
By same tests you mean go tests?

> It would be great if the docker images are tested via CI.

I'm curious what you mean by testing docker images.

**Comment by odidev** [comment]
Sorry, I got confused with the package testing and docker image testing. Actually, I would like to see the same test for the arm64 platform which we are doing for amd64 using the GA.

**Reply by sagikazarmark** [reply]
From a functional perspective, running the test suite on arm64 doesn't provide that much more value. It only catches weird edge cases. So I'd say it's quite safe to use it on arm64 already.
